packages/server/tests/scripts/createAliasAccounts.js

```diff
@@ -0,0 +1,52 @@
+const { Client, PrivateKey, TransferTransaction, AccountId, Hbar } = require('@hashgraph/sdk');
```

Comment:
Should we reposition this file and place it in ./scripts instead of ./package/server/tests/scripts? Just don't wanna have a non-mocha files in the tests folder. Open for discussion tho